app/dao/notifications_dao.py

```diff
@@ -351,7 +351,7 @@ def insert_notification_history_delete_notifications(
         WHERE service_id = :service_id
           AND notification_type = :notification_type
           AND created_at < :timestamp_to_delete_backwards_from
-          AND key_type = 'normal'
+          AND key_type in ('normal', 'team')
```

Comment:
Optional: This is fine logically. Did you do any investigation to performance for using an `in` vs using an `OR`. I wouldn't be surprised if the query plan for the `in` got changed to an `OR` anyway and made no difference but just thought I'd check.